gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -73,20 +77,36 @@ func NewVMKeeper(
 	return vmk
 }
 
-func (vm *VMKeeper) Initialize(ms store.MultiStore) {
+func (vm *VMKeeper) Initialize(logger *slog.Logger, ms store.MultiStore) {
 	if vm.gnoStore != nil {
 		panic("should not happen")
 	}
 	alloc := gno.NewAllocator(maxAllocTx)
 	baseSDKStore := ms.GetStore(vm.baseKey)
 	iavlSDKStore := ms.GetStore(vm.iavlKey)
 	vm.gnoStore = gno.NewStore(alloc, baseSDKStore, iavlSDKStore)
-	vm.gnoStore.SetPackageGetter(vm.getPackage)
 	vm.gnoStore.SetNativeStore(stdlibs.NativeStore)
-	if vm.gnoStore.NumMemPackages() > 0 {
+	if vm.gnoStore.NumMemPackages() == 0 {
```

Comment:
this isn't atomic. alternatively maybe loop through and check each package in order.

Reply:
is it supposed to be?

it's not like we Initialize in places where we are 'concurrent'